vmware_tray/VMWAddOnsTray.cpp

```diff
@@ -360,7 +360,7 @@ VMWAddOnsMenu::VMWAddOnsMenu(VMWAddOnsTray* tray, bool in_vmware)
 
 	AddSeparatorItem();
 
-	AddItem(new BMenuItem("About "APP_NAME B_UTF8_ELLIPSIS, new BMessage(B_ABOUT_REQUESTED)));
+	AddItem(new BMenuItem("About " APP_NAME B_UTF8_ELLIPSIS, new BMessage(B_ABOUT_REQUESTED)));
```

Comment:
Was it intended?

Reply:
Yes, it removes the following warning for a x86_64 build with gcc8:
```
VMWAddOnsTray.cpp:363:24: warning: invalid suffix on literal; C++11 requires a space between literal and string macro [-Wliteral-suffix]
  AddItem(new BMenuItem("About "APP_NAME B_UTF8_ELLIPSIS, new BMessage(B_ABOUT_REQUESTED)));
```
Technically, it's not related to fixing the x86_gcc2 build, so I can move it into another PR if you prefer this one to stay focused on the x86_gcc2 fixes.

Reply:
I was wondering it that would result in double space. I guess not(?), so it's ok.

Reply:
No, there will be just one space. The preprocessor would expand it into the following:
```
"About " "VMW Add-ons" "…"
``` 
And that is concatenated by the compiler into a single string of `"About VMW Add-ons…"`